channels/rdpei/client/rdpei_main.c

```diff
@@ -1310,7 +1330,7 @@ static UINT rdpei_pen_hover_update(RdpeiClientContext* context, INT32 externalId
 	va_start(ap, y);
 	error = rdpei_pen_process(context, externalId,
 	                          RDPINPUT_CONTACT_FLAG_UPDATE | RDPINPUT_CONTACT_FLAG_INRANGE,
-	                          fieldFlags, x, y, FALSE, ap);
+	                          fieldFlags, x, y, ap);
```

Comment:
with the begin logic added to rdpei_pen_process rdpei_pen_hover_update is now the same as rdpei_pen_hover_begin so could delete one

Reply:
true, but lets keep it the way it is.